app/src/layouts/map/maps.vue

```diff
@@ -457,7 +433,7 @@ export default defineComponent({
 					geojsonLoading.value = true;
 					geojsonProgress.value = 0;
 					geojsonError.value = null;
-					geojson.value = await workerProxy(items.value, _layoutOptions.value, proxy(onProgress));
+					geojson.value = await workerProxy(items.value, _layoutOptions.value, '', proxy(onProgress));
```

Comment:
Not sure about this..!

Reply:
Haha, I was just testing how to make custom templates work. 😄